packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -610,11 +610,15 @@ class SliverReorderableListState extends State<SliverReorderableList> with Ticke
       _autoScroller?.stopAutoScroll();
       _autoScroller = EdgeDraggingAutoScroller(
         _scrollable,
-        onScrollViewScrolled: _handleScrollableAutoScrolled
+        onScrollViewScrolled: _handleScrollableAutoScrolled,
+        velocityScalar: _kDefaultSelectToScrollVelocityScalar
       );
     }
   }
 
+  // An eye-balled value for a smooth scrolling speed.
+  static const double _kDefaultSelectToScrollVelocityScalar = 30;
```

Comment:
This does not allow the user of ReorderableList/SliverReorderableList to change this value. Can you expose it on those classes so it can be customized?

Reply:
Thanks for your review! I changed the code so please check it. I added tests too but they fail and I'm not sure if my understanding is correct....